viite-UI/src/model/RoadCollection.js

```diff
@@ -181,10 +181,13 @@
         });
       });
 
-      [unaddressedUnderConstructionRoadLinkGroups, unaddressedUnknownRoadLinkGroups] = _.partition(fetchedUnaddressed, function (group) {
```

Comment:
eikös tämän voi vaihtoehtoisesti tehdä 
(unaddressedUnderConstructionRoadLinkGroups, unaddressedUnknownRoadLinkGroups) = _.partition(...)

Reply:
kyseessä on javascript eikä scala, omaa huolimattomuuttani

Reply:
kyl toi const (jotakin0,jotakin1) = object vois toimia mut ei ehkä listassa,
taitaa toimia myös listoissa
https://www.freecodecamp.org/news/array-destructuring-in-es6-30e398f21d10/

Reply:
Bugin syy oli siinä että "unaddressedRoadLinkGroups" lista oli jäänyt päivittämättä kun refaktoroin aikaisemmin tuon partitioimaan suoraan noihin kahteen listaan. 

Eli sieltä oli jäänyt tuo "unaddressedRoadLinkGroups" listan päivitys välistä pois ja ongelma propagoitui sieltä sitten lopulta bugiksi.